lib/tests-from-pull-request.js

```diff
@@ -1,32 +1,38 @@
-const fs = require('fs')
-const readline = require('readline')
+module.exports = getTestsFromPR;
 
-module.exports = getTestsFromPR
-
-function getTestsFromPR (context) {
+function getTestsFromPR(context) {
   return Promise.resolve()
-    .then(() => ['Hi Or'])
-  // .then(() => _getTestsFromPR(context))
+    .then(() => _getTestsFromPR(context));
 }
 
-function _getTestsFromPR (context) {
-  const pull = context.issue()
-  const {repos, pullRequests} = context.github
-  // todo: code to get file contect
-  let newTestLines = []
-  let PRfiles
+function _getTestsFromPR(context) {
+  let PRfiles;
+  const pull = context.issue();
+  const {pullRequests} = context.github;
+
   return pullRequests.getFiles(pull)
     .then((res) => res.data)
     .then((files) => PRfiles = files)
```

Comment:
the only place you use PRfiles is in the next line. So you can remove the `let PRfiles` declaration and pass the results straight to the next line instead